tests/basictests/dsp-operator.sh

```diff
@@ -56,7 +56,7 @@ function create_pipeline() {
 
     ROUTE=$(oc get route ds-pipeline-ui-sample --template={{.spec.host}})
     SA_TOKEN=$(oc create token ds-pipeline-ui-sample)
-    PIPELINE_ID=$(curl -s -k -H "Authorization: Bearer ${SA_TOKEN}" -F "uploadfile=@${RESOURCEDIR}/test-pipeline-run.yaml" https://${ROUTE}/apis/v1beta1/pipelines/upload | jq -r .id)
```

Comment:
Ideally should work even without the quotes
https://github.com/opendatahub-io/data-science-pipelines/blob/master/tests/basictests/ds-pipelines.sh#L40 for instance